### PR TITLE
Replace colors by ansis

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
You are using the `colors` package that is installed by the MagicMirror² core. [We are about to remove it from the core](https://github.com/MagicMirrorOrg/MagicMirror/pull/3350/files), then your module would have a problem with this dependency starting with the next MagicMirror² release.

Instead of including `colors` into your dependencies, I think it would be a better approach to replace it with the drop-in replacement `ansis` (as we about to do in core).

I had problems getting your build process to run, so I haven't tested my changes. But I am positive that it will work :slightly_smiling_face: 

In the second commit I also made a few cosmetic improvements.

### CODE REVIEW

**Comment by djey47** [comment]
Hello, colors is only used within a 'mock' server faking some APIs for development purposes.

I guess I should have added colors to my own package.json at the first place. It just worked as MM2 is providing it, I wasn't aware of it.

Okay to use ansis as alternative.

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "dev-server": "nodemon --watch ./tools/dev-server ./tools/dev-server/dev-server.js"
   },
   "dependencies": {
+    "ansis": "^2.0.3",
```
Comment (djey47) [inline package.json]: I guess it's only required as a devDependency,  since I don't use it within the node helper, only in a development tool.

**Review by djey47 — CHANGES_REQUESTED** [review]

**Comment by KristjanESPERANTO** [comment]
> I guess I should have added colors to my own package.json at the first place.

Exactly :slightly_smiling_face:

I've searched through all the modules on the official list so that we don't break any modules with this change for the next release.

File: package.json (hunk 1 above)
Comment (KristjanESPERANTO) [reply package.json]: Okay, moved it to devDependencies.

**Comment by djey47** [comment]
Thanks. But are you doing this for each one of the MM2 modules? That's quite heavy work !

And concerning the CI, indeed the master branch has issues breaking the workflow (unit tests), it should be sorted out after merging my current dev branch (transilien) which is running well.

**Comment by KristjanESPERANTO** [comment]
> Thanks. But are you doing this for each one of the MM2 modules? That's quite heavy work !

I am in the process of building [a new list of 3rd party modules](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/) based on the official list. I'm already checking a few rudimentary things on all modules. So I just used this to search for `require('colors` :smile:

**Comment by KristjanESPERANTO** [comment]
I'm not that familiar with your build process, so I don't know why your tests are failing. Can you handle it? Otherwise we can just add colors to your package.json.

**Comment by djey47** [comment]
The issue might be related to old libraries I've removed by changing the APIs I'm using. It is absolutely not related to the removal of colors as it already failed before.

I'm positive that merging my dev branch will eventually get everything running great with the latest changes. Btw, I will merge upcoming master into it to check.

Anayway, thanks for your help proposal.

**Comment by djey47** [comment]
Created a new PR with the same changes after latest master update: https://github.com/djey47/MMM-IDF-LOCALTRANSPORT/pull/18

**Pull request closed without merging**